Lombiq.VueJs.Resources/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "private": true,
+  "dependencies": {
+    "vue": "3.5.22",
+    "vue-router": "4.6.3"
```

Comment:
These are added by `Lombiq.VueJs` too. Why not make that project depend on this one? The config in `ResourceManagementOptionsConfiguration` is duplicated too.

Reply:
Intentionally. My thought is that we will update major and minor versions here, but only patch versions in `Lombiq.VueJs` since that will be deprecated. If a future version breaks LV, then we can diverge without headaches.